src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,9 @@
-export const PostList = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts }) => (
+  <>
```

Comment:
fix it everywhere

Reply:
try to read where do we need to use this fragment
https://react.dev/reference/react/Fragment